File: api-tester/src/main/java/com/revenuecat/apitester/java/EntitlementVerificationModeAPI.java
package com.revenuecat.apitester.java;

import com.revenuecat.purchases.EntitlementVerificationMode;

@SuppressWarnings({"unused"})
final class EntitlementVerificationModeAPI {
    static void check(final EntitlementVerificationMode verificationMode) {
        switch (verificationMode) {
            case DISABLED:
            case INFORMATIONAL:
            // Hidden ENFORCED mode during feature beta
            // case ENFORCED:
        }
    }
}

File: api-tester/src/main/java/com/revenuecat/apitester/kotlin/EntitlementVerificationModeAPI.kt
package com.revenuecat.apitester.kotlin

import com.revenuecat.purchases.EntitlementVerificationMode

@Suppress("unused", "UNUSED_VARIABLE")
private class EntitlementVerificationModeAPI {
    fun check(verificationMode: EntitlementVerificationMode) {
        when (verificationMode) {
            EntitlementVerificationMode.DISABLED,
            EntitlementVerificationMode.INFORMATIONAL,
            // Hidden ENFORCED mode during feature beta
            // EntitlementVerificationMode.ENFORCED
            -> {}
        }.exhaustive
    }
}
